palmr/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: palmr
+category: files
+name: Palmr
+version: "3.2.5-beta"
+tagline: Palmr. - Open-Source File Transfer
```

I would get rid of the app name here.
```suggestion
tagline: Open-Source File Transfer
```